### DISCUSSION TITLE
Sunbird Ed environments availability

### DISCUSSION THREAD

**Discussion by gandham-santhosh** [Sunbird-Ed/Community#610] [Announcements]
Hi Team, 
As part of our ongoing efforts to optimize the infrastructure costs for the Sunbird Ed environment, we have conducted a thorough review of the usage data from the past few months. Based on our analysis, we have observed that the environments remain inactive during weekends, public holidays and after 9:00 pm and before 9:00 am.

Taking this into consideration, we have decided to implement a new operational schedule. Starting immediately, the Sunbird Ed environment will be temporarily shut down during weekends and public holidays. This adjustment will allow us to maximize cost efficiency while ensuring that resources are allocated optimally. During the regular working days, the environments will be available for use from 9 AM to 9 PM.

If you encounter any difficulties or have specific concerns regarding this schedule adjustment, please don't hesitate to reach out to me directly via email at santhosh.g@optit.in. Your support and collaboration are essential as we continue to enhance the efficiency of our infrastructure. Thank you

@vinukumar-vs @aks30 @rajeevsathish  @AmiableAnil  @sajeshkayyath @manjudr @pallakartheekreddy @maheshkumargangula @mohitga @Krishnaj20 @chitranshu-keshav  @madhucr @SanthoshVasabhaktula @vrayulu

**Comment by gandham-santhosh** [comment]
@kiranharidas187 @Prateek-slokam@parthlawate @baljeetsingh @manojLondhe

**Reply by kiranharidas187** [reply]
@gandham-santhosh  just to confirm, this shutdown will not happen for optional holidays right? Asking since optional holidays are different for different organisations.
